cmd/exporter.go

```diff
@@ -73,13 +82,14 @@ func main() {
 	}
 
 	if modelServerEndpoint != nil {
-		model.SetModelServerEndpoint(*modelServerEndpoint)
+		config.SetModelServerEndpoint(*modelServerEndpoint)
 	}
 
 	newCollector, err := collector.New()
 	defer newCollector.Destroy()
 	defer rapl.StopPower()
 	if err != nil {
+		finalizer()
```

Comment:
Isn't the log enough? The log will already print a string....

In the case we need the function, the `finalizer()` looks like something that will run when the program exit, so we could defer as the first line of the main

Reply:
Agree. Just does not come out with other function name.